core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/common/ResourceStatus.java

```diff
@@ -4,5 +4,13 @@
     DEFAULT,
     DEFAULT_DELETED,
     USER_MANAGED,
-    OUTDATED
+    OUTDATED;
```

Comment:
Do we need this? It is going to break during CB upgrade.

Reply:
I just put a ; sign at the end